app/domain/authentication/authn_k8s/k8s_object_lookup.rb

```diff
@@ -25,6 +25,10 @@ def initialize(webservice = nil)
         @cert_store = OpenSSL::X509::Store.new
         @cert_store.set_default_paths
         ::Conjur::CertUtils.add_chained_cert(@cert_store, ca_cert)
+
+        return unless ENV.key?('SSL_CERT_DIRECTORY')
+
+        load_additional_certs(ENV['SSL_CERT_DIRECTORY'])
```

Comment:
I might be missing something but is the singular call to `add_chained_cert` above not enough ? Wouldn't all the certs be stored in the Conjur variable or the cert file on disk when running inside a pod ?

Separately, I'm wondering about the documentation around the envvar `SSL_CERT_DIRECTORY`, I'm not familiar with it yet I see it being used in a few repos and without looking at the code here I wouldn't immediately know how it is used... and I'm not sure if its meaning in the context of the code here is shared across the different places it is used.